### ISSUE TITLE
Rdpwrap didn't work

### ISSUE THREAD

**Issue by moreleb** [stascorp/rdpwrap#594]
Hello,

I am asking You for help, RDPWrap has stopped working after updating Windows 10. By logging into the 2nd account in the system, 1 user automatically logs out (without any questions).

![rdp](https://user-images.githubusercontent.com/44367715/47767783-83ab8180-dcd5-11e8-8606-11ad22a4a088.jpg)
![win10](https://user-images.githubusercontent.com/44367715/47767786-860ddb80-dcd5-11e8-8d88-e87420a241b4.jpg)

**Comment by millsysbl** [comment]
I've had luck uninstalling RDPWrap, reboot, then reinstall

**Comment by moreleb** [comment]
I tried, unfortunately it does not work.

**Comment by millsysbl** [comment]
Try another Windows update, 16299 is older

**Comment by moreleb** [comment]
Windows Update can not find newer updates. The system claims it is up to date.

**Comment by millsysbl** [comment]
https://drive.google.com/open?id=1XGErWqbPTRvLuRMtXUOKmG07-IxSFNX-

extract
run sysmate system file walker
first row is c:\windows\system32\termserv.dll
second is the location you extracted the 16299 version
click start, force it, then restart remote desktop service

you won't need or want RDP wrapper for this

**Comment by moreleb** [comment]
> _Using Method 1]
> Succeeded taking ownership of: "C:\Windows\System32\termsrv.dll"
> Flushing cache...
> [Using Method 1]
> Processing permission on: "C:\Windows\System32\termsrv.dll" failed.
> Operations failed._

I have full permissions to the termsrv.dll file.
I used a different option - I turned off the remote desktop service - I deleted the termsrv.dll file - I copied your termsrv.dll file - I turned on the service.

Did not help.

**Comment by moreleb** [comment]
I have uninstalled RDPWrap.
I have replaced the termsrv.dll file
I have changed the gpedit settings
I have changed the registry settings

It still does not work. By logging in from a remote account, I have no questions that 1 user is logged in. Simply, it looks like I have switched the user.

Interestingly, 1 user is not logged out (does not see the lock screen) he sees what 2 user does remotely

**Comment by moreleb** [comment]
I made a check of sfc / scannow system files and after that, during a remote connection, a message about the logged-in user appears.

I replaced the termsrv.dll file with the one given in the post - and the same: by logging in I take control of the logged in local account.

**Comment by moreleb** [comment]
I managed to manually update the system to version 10.0.17134 (17134.376). I need a new termsrv.dll file

**Comment by millsysbl** [comment]
Try RDPWrap again should work with that version.  Run the installer as
administrator

**Comment by moreleb** [comment]
It works.
I found a file with the appropriate version along with a nice conversion script.

https://drive.google.com/file/d/1R_9mGaeM3zNSnZuCuZNmox2-px4O75VN/view?usp=sharing

I've tried everything. I have reset Group Policy and Local Security Policies settings to default settings - it did not help either. The problem was that everything was done remotely using the Team Viewer v13 session.

What I did:
- in 1 main account I enabled automatic login without entering a password. (Start-> Run-> control userpasswords2)
- i connected remotely with 1 account via RDP and uninstalled TeamViewer v13. (when disconnecting TeamViewer sessions, system logged me out on the RDP session)
- restart the computer
- I connected via RDP (with 1 account) - changed termsrv.dll files (script)

In addition, I installed TeamViewer Host version 9.
Now it does not matter if I'm connected to 1 account via TeamViewer or RDP, when I connect to the second account, it does not log me out
